### ISSUE TITLE
OpenCV viz crashes once execution starts in VS2015

### ISSUE THREAD

**Issue by kelray** [UnaNancyOwen/VelodyneCapture#30]
Hello Tsukasa,

I'm encountering an issue with the "Viewer" example when compiling it with Visual Studio 2015 (x64), the executable crashes with this message:
<img width="960" alt="abort_issue" src="https://user-images.githubusercontent.com/8460504/47597332-b3a7fd00-d942-11e8-8f4d-287baf0d38d4.png">

After some code tracing, the issue appears to be in these two lines of code:
`cv::Mat cloudMat = cv::Mat(static_cast<int>(buffer.size()), 1, CV_32FC3, &buffer[0]);
cv::viz::WCloud cloud(cloudMat, viz::Color::white());`

But haven't had any luck resolving the issue, have you faced this issue before?.

Thanks,
Karim

**Comment by UnaNancyOwen** [comment]
It is different from the original code.
The original code is following.

```cpp
cv::Mat cloudMat = cv::Mat( static_cast<int>( buffer.size() ), 1, CV_32FC3, &buffer[0] );
cv::viz::WCloud cloud( cloudMat );
```

Also, You specify white (<code>viz::Color::white())</code>) but display of attached image is green. Why?
At the least, You need to tell the OpenCV version.

**Comment by kelray** [comment]
You are right, I modified this part after I took the screenshot, however that doesn't change anything, even if I remove the color option still it crashes, I tried all options.
I'm using OpenCV v.3.4.3.

**Comment by kelray** [comment]
Any luck with this issue?, I had very little progress trying to solve it, so, now it doesn't crash when I replaced `viewer.spinOnce();` with `viewer.spin();`, However the application doesn't loop and stuck at the first frame constructed, this is a screenshot:
<img width="482" alt="opencv_viz_sample" src="https://user-images.githubusercontent.com/8460504/48809504-bbf11f00-ecd9-11e8-8c0a-525235f3539d.png">
I tried to print out the values of x, y and z, it stops after 31935 readings (the pcap file has 8305 data points. Just in case you are wondering what is the VTK version I have used when building Viz, it's v8.1.
